source/_docs/installation/synology.markdown

```diff
@@ -2,78 +2,287 @@
+</p>
+
+<p class="note">
+Update: You can install a more recent home assistant (eg, 0.84.2) by manually compiling a python3 package using the <a href="https://github.com/SynoCommunity/spksrc">spksrc compilation framework</a>. The new installation guide will be a little different though, but the upside is most failing components will now work, such as “<a href="/components/discovery/">Discovery</a>”, “<a href="/components/cloud/">Cloud</a>” and “<a href="/components/Xiaomi_Aqara/">Xiaomi_Aqara</a>”. Thread for more info: <a href="https://community.home-assistant.io/t/python-3-5-3-on-synology/46372/27">Python >=3.5.3 on Synology</a>.
 </p>
 
 There are 2 alternatives, when using Home Assistant on Synology NAS:
-1. using Docker
-2. directly running on DSM
+1. Using Docker
+2. Directly running on DSM
 
 Option 1 is described on the [Docker installation page](/docs/installation/docker/), whereas Option 2 is described below.
 
 
-The following configuration has been tested on Synology 413j running DSM 6.0-7321 Update 1.
+The following configuration has been tested on [Synology DS115j](https://www.synology.com/en-global/products/DS115j) running DSM 6.2.1-23824 Update 1.
 
 Running these commands will:
+* Compile the required new Python3 package
+* Install Home Assistant
+* Enable Home Assistant to be launched on http://*Synology_IP*:8123
+
+Using the [Synology webadmin](https://www.synology.com/en-global/knowledgebase/DSM):
+* Install compiled Python3 package using the [Synology Package Center](https://www.synology.com/en-global/knowledgebase/DSM/help/DSM/PkgManApp/PackageCenter_desc)
+* Create “homeassistant” user and add to the “users” group
+* Create “homeassistant” “Shared-Folder”
+* Optionally Start Home Assistant on Bootup of Synology
 
- - Install Home Assistant
- - Enable Home Assistant to be launched on [http://localhost:8123](http://localhost:8123)
 
-Using the Synology webadmin:
+Like you might have read above, at this time of writing, Synology has not provided a recent [Python3](https://www.synology.com/nl-nl/dsm/packages/py3k) package, so we have to create a recent Python3 package ourself.
+Do not worry though, were gonna try keep this as painless as possible.
 
- - Install python3 using the Synology Package Center
- - Create homeassistant user and add to the "users" group
+## {% linkable_title Preparing prerequisites %}
 
-SSH onto your synology & login as admin or root
+In this guide we are going to use a Linux based Operating System such as [Ubuntu](https://www.ubuntu.com/), [Mint](https://linuxmint.com/) or [Solus](https://getsol.us) to minimize difficulty. You will also need [Docker](https://docs.docker.com/install/).
+You can think of [Docker](https://docs.docker.com/install/) as a way more powerful [Chroot](https://help.ubuntu.com/community/BasicChroot) and containers as ready made mini operating systems.
```

Comment:
I think that this is nice to know but doesn't add value to the guide.

Reply:
I am assuming Windows users and people on different Linux distributions might follow the guide, so it is nice to let them know what system they could run to make everything work nicely.
I could remove the Docker sentence though.

Reply:
The steps displaying how to install Docker already has a header for the system it runs on.
Currently the header for Ubuntu, Mint and derivatives has a hyperlink to the Docker page and the Solus URL to the Solus homepage.
For Solus there is no specific Docker page.
Should I change the Ubuntu hyperlink to the appropriate homepage hyperlink, or leave as is?

Reply:
I have changed as you requested and removed that part, should I change the header to instead include the homepages, leave as Docker installation page hyperlink or no hyperlink at all?